web/layouts/settings-general/tab.html

```
@@ -1,4 +1,8 @@
+<div class="pc-settings-shell pc-content-card flex w-full flex-1 flex-col lg:flex-row">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)